src/App.css

```diff
@@ -1 +1,21 @@
 /* not empty */
+.pagination {
+  width: 20%;
+}
+
+.page-item{
+  padding-inline: 10px;
+  border: 1px solid grey;
+}
```

Comment:
Consider using a more specific selector for `.page-item` to avoid potential conflicts with other elements that might have the same class. If `.page-item` is only used within `.pagination`, you could nest it under `.pagination` in your CSS.